nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1003,7 +202,7 @@ in
       type = with types; nullOr path;
       default = null;
       example = "/var/lib/hedgedoc/hedgedoc.env";
-      description = lib.mdDoc ''
+      description = mdDoc ''
         Environment file as defined in {manpage}`systemd.exec(5)`.
 
         Secrets may be passed to the service without adding them to the world-readable
```

Comment:
Might as well also migrate from `EnvironmentFile` to `LoadCredential`
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Credentials
https://systemd.io/CREDENTIALS/

Reply:
I guess I got caught by the description mentioning secrets provisioning, which is in general better done using `LoadCredential`. However, after studying the documentation of Hedgedoc on how to configure it, I no longer believe that `LoadCredential` is a good fit here.